frontend/uddhar-frontend/src/shared/components/DashboardNavbar.jsx

```
@@ -64,7 +64,7 @@ const DashboardNavbar = ({ children, heading }) => {
             {userMenuOpen && (
               <ul className="absolute right-0 mt-2 w-40 bg-white border rounded-lg z-10 list-none flex flex-col">
                 <li>
-                  <button className="w-full text-left p-2 text-gray-700 hover:bg-blue-200 cursor-pointer">
+                  <button className="w-full text-left p-2 text-gray-700 hover:bg-blue-200 cursor-pointer" onClick={()=>navigate("/profile")}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)